docs/document/content/faq/_index.en.md

```diff
@@ -230,7 +230,7 @@ The solutions are as follows:
 Answer:
 
 1. Update to 4.0.1 above, which helps speed up the process of loading table metadata from `the default dataSource`.
```

Comment:
However, `.` still workS well for 4.x release. `-` is for the upcoming 5.x. I suppose more clarification is required.